packages/mongodb-chatbot-server/src/services/ConversationsService.ts

```diff
@@ -211,6 +224,15 @@ export interface ConversationsService {
     messageId,
     rating,
   }: RateMessageParams) => Promise<boolean>;
+
+  /**
+    Add a user comment to a {@link Message} that already has a rating.
```

Comment:
nit: msg doesn't need a rating to add a comment

 ```suggestion
    Add a user comment to a {@link Message}.
```

Reply:
Hmm yeah that's reasonable I can look to adjust it